static/bidder-params/appnexus.json

```diff
@@ -5,14 +5,22 @@
 
   "type": "object",
   "properties": {
-    "placementId": {
+    "placement_id": {
       "type": "integer",
       "description": "An ID which identifies this placement of the impression"
     },
-    "invCode": {
+    "placementId": {
+      "type": "integer",
+      "description": "Deprecated"
```

Comment:
I know it's fairly obvious but maybe add `"description": "Deprecated; use placement_id instead."

Reply:
I can certainly do that.